lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -173,7 +173,7 @@ export class NativeInteractionClient extends BaseInteractionClient {
         // fall back to native calls
         const messageBody: NativeExtensionRequestBody = {
             method: NativeExtensionMethod.GetToken,
-            request: nativeRequest,
+            request: { ...nativeRequest, keyId: "" },
```

Comment:
Why are we passing a blank keyId here? Can we simply omit it?

Reply:
We are intentionally blanking it out because it contains a value when you initialize the request but it is not required in the native token request to acquire tokens. It is required a validation step while caching the tokens later.

Reply:
I don't thing we need to blank it unless the native flow crashes with it. WAM should just ignore it if it doesn't need it right?

Reply:
Setting the value to empty string still includes the param on the request. If not including it at all is the desired outcome it should be done like: `const { ...restNativeRequest, keyId} = nativeRequest;` and then pass `restNativeRequest` into the messageBody

FWIW I think there's other irrelevant params being sent to the broker today that could be omitted as well if this is something we're trying to do.

Reply:
Addressed!

Reply:
General comment - we should avoid passing parameters to the broker that it ignores today.  This is limiting if a future version of the broker needs to then inspect these values.  Especially since the broker can be updated without updates to the JS/Broker pathway.